### PR TITLE
codet5-base.jsoup.mid-1408.idx-60631.3.mutant

### PR DESCRIPTION
**Pull request by ahmedfahad04**
fix indentation isDoctype() - Token.java

### CODE REVIEW

File: jsoup/src/main/java/org/jsoup/parser/Token.java [hunk 1]
```diff
@@ -443,9 +443,9 @@ public String toString() {
     }
 
     final boolean isDoctype() {
-            if(type == TokenType.Doctype)
-                    return true;
-            return false;
+        if(type == TokenType.Doctype)
+                return true;
```
Comment (faustinoaq) [inline jsoup/src/main/java/org/jsoup/parser/Token.java]: Needs to fix indentation to match formatting 8 spaces -> 4 spaces

**Review by faustinoaq — CHANGES_REQUESTED** [review]

File: jsoup/src/main/java/org/jsoup/parser/Token.java (hunk 1 above)
Comment (darko-marinov) [reply jsoup/src/main/java/org/jsoup/parser/Token.java]: @ahmedfahad04 please address this in the next 2-3 days, or we'll assign to someone else.

Comment (ahmedfahad04) [reply jsoup/src/main/java/org/jsoup/parser/Token.java]: Fixed the indentation ✅

**Comment by darko-marinov** [comment]
Space after `if` is missing, and curly braces may be also missing, but still accepting.

**Pull request merged**